pkgs/applications/terminal-emulators/mlterm/default.nix

```
@@ -28,20 +28,20 @@ stdenv.mkDerivation rec {
     gtk3
     harfbuzz
     fribidi
+    vte
   ] ++ lib.optionals (!stdenv.isDarwin) [
     # need linker magic, not adapted for Darwin yet
     openssl
     libssh2
 
     # Not supported on Darwin
-    vte
     m17n_lib
 
     fcitx5
     fcitx5-gtk
     ibus
-  ] ++ lib.optionals (stdenv.system != "aarch64-linux") [
-    # FIXME Currently broken on aarch64-linux
+  ] ++ lib.optionals (!stdenv.isAarch64 && !stdenv.isDarwin) [
```

We could just update `uim.meta.broken` and then

```suggestion
  ] ++ lib.optionals (!uim.meta.broken) [
```

---

Also, `uim` actually compiles on `aarch64-linux`...